Makefile

```diff
@@ -30,6 +30,17 @@ test: $(SOURCES)
 lint: $(SOURCES)
 	$(DEPSPATH)/golangci-lint run
 
+.PHONY: mdlint
+# Install it locally: https://github.com/igorshubovych/markdownlint-cli#installation
+# Or see `mdlint-ctr` below or https://github.com/DavidAnson/markdownlint#related.
+mdlint:
+	markdownlint '**/*.md'
+
+.PHONY: mdlint-ctr
+# If markdownlint is not installed, you can run markdownlint within a container.
+mdlint-ctr:
+	docker run --rm -v "$(shell pwd):/repo:ro" -w /repo avtodev/markdown-lint:v1 '**/*.md'
+
 .get-deps-stamp:
```

Comment:
do you want to add `markdownlint` as a dep here?

Reply:
I got the idea of packaging deps in `get-deps-stamp`. But from the documentation, the recommended way is by [npm](https://github.com/igorshubovych/markdownlint-cli#installation). They will run into failure if they have not set up npm in their environment, and it might be too heavy-handed to add npm installation steps in `get-deps-stamp`. (There are actually multiple ways of installing npm and they can conflict with each other)